api/caching/tasks.py

```diff
@@ -94,3 +99,17 @@ def ban_url(instance):
                     logger.info('Banning {} succeeded'.format(
                         url_to_ban,
                     ))
+
+
+@app.task(max_retries=5, default_retry_delay=10)
```

Comment:
These numbers are arbitrary.

Reply:
Have we thought about what to do if the cache is out of sync?  For example, I upload a 5GB file, but this task never completes, and my project continues to say I'm not using any storage.

Reply:
@Johnetordoff, thoughts? What about an alert if a task doesn't complete?  A script to rerun all storage calculations is probably too time-consuming.

Reply:
Well it will totally recalculate the next time it's updated so if you upload a 5GB file and it times out it will not update until you upload another file. If we want to make sure it will eventually be updated even without uploading/copying/deleting a file, we could give the cache an expiration date* to ensure it doesn't get stuck for more than 24 hours or something, but that could be a significant performance penalty on page load for a really large project. I don't know if that's worth it, but it counldn't hurt to have a REALLY long (like a week) expiration rate so nothing is permanently stuck.

So short answer we don't really need a script to deal with this, but it would hurt performance on page project load which we want to avoid as much as possible, at least initially. 

* Right now caches updated for storage usage never expire.

Reply:
yeah, I see what you're saying.  There are tradeoffs between caching this information and pulling it out of the db, but I understand that speed is very much a priority here.  But, this means there are going to be times when our cache is out of sync with the db.  No need to change anything here, but we can monitor and see if/how much of an issue this becomes.